### PR TITLE
Fix taxonomy links

### PR DESCRIPTION
**Pull request by nastasi-oq**
Tests are green here: https://ci.openquake.org/job/zdevel_oq-platform/476/
and here: https://ci.openquake.org/job/zdevel_oq-platform-standalone/184/

### CODE REVIEW

File: openquakeplatform_taxtweb/bin/check_taxonomy_links.sh [hunk 1]
```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+ddown_items="$(grep push $(find . -name '*.js') | grep dataGemHelp | sed "s/.*gem_taxonomy_base + '//g;s/'.*//g" | sort | uniq)"
+
+# 'direction-x', 'direction-y'
+# from views.py:  sub1help = [taxonomy_base + 'direction-x', taxonomy_base + 'direction-y']
```
Comment (hascar) [inline openquakeplatform_taxtweb/bin/check_taxonomy_links.sh:5]: Please delete commented code

File: openquakeplatform_taxtweb/test/click_and_help_test.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+
+        # click_and_help_tag = pla.xpath_finduniq(
+        #     "//li[@id='id_help_form']")
+        # click_and_help_tag.click()
+
+        # simple_help = pla.xpath_finduniq(
+        #     "//li[span[normalize-space(text()) = 'Direction X']]")
+        # simple_help.click()
+
```
Comment (hascar) [inline openquakeplatform_taxtweb/test/click_and_help_test.py]: Please delete commented code

File: openquakeplatform_taxtweb/bin/check_taxonomy_links.sh (hunk 1 above)
Comment (nastasi-oq) [reply openquakeplatform_taxtweb/bin/check_taxonomy_links.sh:5]: This is a reference where direction x and y are defined, not commented code

**Review by hascar — APPROVED** [review]
LGTM

**Pull request merged**